packages/conan/recipes/python/conanfile.py

```diff
@@ -127,9 +126,11 @@ def package(self):
             }
         ):
             self.run(f"{py_exe} get-pip.py")
-            self.run(
-                f"{py_exe} -m pip install nose coverage docutils epydoc numpy=={os.environ['ASWF_NUMPY_VERSION']}"
-            )
+            self.run(f"{py_exe} -m pip install nose coverage docutils epydoc")
+            if self.options.get_safe("with_numpy"):
+                self.run(
+                    f"{py_exe} -m pip install numpy=={os.environ['ASWF_NUMPY_VERSION']}"
+                )
```

Comment:
If this is being done as a container is being built, would it be better to build a single command line to install all the packages from 'pip' at the same time (with optional nympy) to avoid creating an extra container layer?

Reply:
in this case the whole conan build is done as a single command so it won't make a difference. And more interestingly with Conan the package builds are run from a prebuilt docker container, instead of as part of the docker image build, which is closer to what happens in client CI use cases.